bundles/net.tourbook/src/net/tourbook/ui/tourChart/ChartLayerNight.java

```diff
@@ -45,19 +46,23 @@ public ChartLayerNight() {
    @Override
    public void draw(final GC gc, final GraphDrawingData drawingData, final Chart chart, final PixelConverter pc) {
```

Comment:
The photo markers are painted with the night section color

![mt chart photo marker](https://user-images.githubusercontent.com/1283445/115883936-a2c29e80-a44e-11eb-913d-bd1b7c821510.png)

![mt chart photo marker 2](https://user-images.githubusercontent.com/1283445/115884425-2f6d5c80-a44f-11eb-8b3e-d7ac871f0e69.png)

Reply:
I don't use the photo feature so that might be why it slipped through the cracks. I will fix in my next PR.

Reply:
Fixed
![image](https://user-images.githubusercontent.com/25235665/116159729-fa127a00-a6ad-11eb-9fbb-0896a4650ee8.png)

Reply:
> I don't use the photo feature so that might be why it slipped through the cracks. I will fix in my next PR.

There are so many features in the tour chart, therefore we have to be carefull when modifying it. When we would have implemented a UI test environment, would it had found this issue?

Reply:
>When we would have implemented a UI test environment, would it had found this issue?

Maybe it would have found it, not sure, I haven't had the chance to do UI testing yet.
I have fixed the issue in this [commit](https://github.com/FJBDev/mytourbook/pull/57/commits/26de6dd949a54748fe025846a3026ae646afe2da)

Reply:
Would it make sense to join the grid checkbox with the spinner in one line (redbox 2) just like I did for the night section (red box 1)
It would also make the slideout smaller

![image](https://user-images.githubusercontent.com/25235665/116480069-ebf36380-a83d-11eb-912b-00353b30c2ea.png)

Reply:
Good idea

Reply:
Thanks, that works now.
But I see a lot of images in the list, does that mean they are not disposed and need to be ?

![image](https://user-images.githubusercontent.com/25235665/116572337-1f2f0480-a8c9-11eb-99dd-1702c52daaef.png)

Reply:
![image](https://user-images.githubusercontent.com/25235665/116577199-7fc04080-a8cd-11eb-9b47-9cd143a03779.png)